README.md

```diff
@@ -0,0 +1,13 @@
+# Teabot Stats
+
+## Install
+
+```
+npm i && npm run build
+```
+
+## Run
+
+```
+python -m SimpleHTTPServer 5000
```

Comment:
Oh, I thought the web server and api servers run separately? (I.e. would it make more sense to put the "run" instructions for `teabot_endpoints` in `teabot_endpoints`?)